### PR TITLE
Minor fix: Contact link, footer, dashboard page

### PR DESCRIPTION
**Pull request by adriantirusli**
In this update I've changed:

- `components/Footer` to move text to center

- `components/Button` and `pages/index.js` to change contact's url

- `pages/reports/[id].js` to change the data fetching from getStaticProps to getServerSideProps

### CODE REVIEW

**Review by faultables — COMMENTED** [review]
LGTM

File: pages/reports/[id].js [hunk 1]
```diff
@@ -146,23 +147,9 @@ const Report = ({ reports }) => {
...
-}
-
-export async function getStaticProps({ params }) {
-  const res = await fetch(
-    `https://aurora-api.koalafy.dev/reports/${params.id}`
-  );
+export async function getServerSideProps(context) {
+  const { id } = context.query;
+  const res = await fetch(`https://aurora-api.koalafy.dev/reports/${id}`);
   const reports = await res.json();
```
[the remark this answers is not in the document]
Comment (adriantirusli) [reply pages/reports/[id].js:153]: Hmmm forgot that one. What r u suggest with that case?

[the remark this answers is not in the document]
Comment (faultable) [reply pages/reports/[id].js:153]: - check res.status/res.statusCode
- if not equal with 200 then show 404 page

[the remark this answers is not in the document]
Comment (adriantirusli) [reply pages/reports/[id].js]: Pls check my last commit

**Review by faultables — APPROVED** [review]
There is strange behavior that our 404 page returns 200 status code, but I guess this is just a minor issue

LGm

**Pull request merged**